### PR TITLE
8316104: Open source several Swing SplitPane and RadioButton related tests 

### PR DESCRIPTION
**Pull request by TejeshR13**
Open sourcing these Swing SplitPane and RadioButton related tests:

javax/swing/JSplitPane/4147653/bug4147653.java
javax/swing/JSplitPane/4870674/bug4870674.java
javax/swing/JRadioButton/4823809/bug4823809.java

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8316104](https://bugs.openjdk.org/browse/JDK-8316104): Open source several Swing SplitPane and RadioButton related tests (**Bug** - P4)

### Reviewers
 * [Alexander Zuev](https://openjdk.org/census#kizune) (@azuev-java - **Reviewer**) ⚠️ Review applies to [4671cd9f](https://git.openjdk.org/jdk/pull/15738/files/4671cd9f97c1e6ec58dd4d0097184447ab68cfbf)
 * [Abhishek Kumar](https://openjdk.org/census#abhiscxk) (@kumarabhi006 - Committer) ⚠️ Review applies to [4671cd9f](https://git.openjdk.org/jdk/pull/15738/files/4671cd9f97c1e6ec58dd4d0097184447ab68cfbf)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/15738/head:pull/15738` \
`$ git checkout pull/15738`

Update a local copy of the PR: \
`$ git checkout pull/15738` \
`$ git pull https://git.openjdk.org/jdk.git pull/15738/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 15738`

View PR using the GUI difftool: \
`$ git pr show -t 15738`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/15738.diff">https://git.openjdk.org/jdk/pull/15738.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/15738#issuecomment-1719111629)

### CODE REVIEW

**Review by azuev-java — APPROVED** [review]
Looks fine

File: test/jdk/javax/swing/JRadioButton/bug4823809.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    private static ButtonUI testUI;
+    private static volatile boolean passed = false;
+    private static JFrame frame;
+    private static Robot robot;
+
+    public static void main(String[] args) throws Exception {
+        try {
+            robot = new Robot();
+            SwingUtilities.invokeAndWait(() -> {
+                frame = new JFrame();
```
Comment (kumarabhi006) [inline test/jdk/javax/swing/JRadioButton/bug4823809.java:52]: Please add frame title.

**Comment by kumarabhi006** [comment]
Do we need to add auto delay also for the tests? 
`robot.setAutoDelay(100);`

File: test/jdk/javax/swing/JRadioButton/bug4823809.java (hunk 1 above)
Comment (TejeshR13) [reply test/jdk/javax/swing/JRadioButton/bug4823809.java:52]: It wont be visible, no point in adding title.

**Comment by TejeshR13** [comment]
> Do we need to add auto delay also for the tests? `robot.setAutoDelay(100);`

I don't think its mandatory, no keyboard events are there here.

File: test/jdk/javax/swing/JRadioButton/bug4823809.java (hunk 1 above)
Comment (kumarabhi006) [reply test/jdk/javax/swing/JRadioButton/bug4823809.java:52]: I think to maintain consistency across tests we should add title. Many of the tests were not visible but still frame title added.

**Review by kumarabhi006 — APPROVED** [review]

File: test/jdk/javax/swing/JRadioButton/bug4823809.java (hunk 1 above)
Comment (TejeshR13) [reply test/jdk/javax/swing/JRadioButton/bug4823809.java:52]: Will add it.

**Pull request closed without merging**